packages/xo-server/src/xo-mixins/rest-api.mjs

```diff
@@ -302,6 +305,45 @@ async function _getDashboardStats(app) {
     return false
   }
 
+  function _extractVmIdsFromBackupJob(job) {
+    let vmIds
+    try {
+      vmIds = extractIdsFromSimplePattern(job.vms).filter(vmId => app.hasObject(vmId, 'VM'))
+    } catch (_) {
+      const predicate = createPredicate(job.vms)
+      vmIds = nonReplicaVms.filter(predicate).map(vm => vm.id)
+    }
+    return vmIds
+  }
+
+  function _updateVmProtection(vmId, isProtected) {
+    if (vmIdsProtected.has(vmId)) {
+      return
+    }
+
+    const vm = app.getObject(vmId, 'VM')
+    if (vmContainsNoBakTag(vm)) {
```

Comment:
so the VM will not be marked as protected if at least one of its disk has NoBak tag. Is it clear in the UX
For me protected means : it is protected as much as the user want it to, si I would not excludes theses VM

Reply:
My bad : this is about the no bak on the vm